data/json/recipes/armor/torso.json

```
@@ -224,12 +254,21 @@
     "subcategory": "CSC_ARMOR_TORSO",
     "skill_used": "fabrication",
     "difficulty": 8,
-    "time": "3 h 10 m",
+    "time": "9 h",
     "book_learn": [ [ "textbook_armwest", 7 ] ],
     "using": [ [ "blacksmithing_standard", 32 ], [ "steel_standard", 8 ] ],
     "qualities": [ { "id": "CHISEL", "level": 3 } ],
     "tools": [ [ [ "swage", -1 ] ] ],
-    "components": [ [ [ "fur", 6 ], [ "tanned_pelt", 1 ], [ "leather", 6 ], [ "tanned_hide", 1 ] ] ]
+    "components": [ [ [ "fur", 6 ], [ "tanned_pelt", 1 ], [ "leather", 6 ], [ "tanned_hide", 1 ] ] ],
+    "proficiencies": [
+      { "proficiency": "prof_armorsmithing", "required": false, "time_multiplier": 2 },
+      {
+        "proficiency": "prof_blacksmithing",
+        "required": false,
+        "learning_time_multiplier": 0.5,
+        "time_multiplier": 1.15
```

```suggestion
        "time_multiplier": 3
```

Blacksmithing 3, armorsmithing 2.5 seems to be the general balance we've landed on